google-cloud-datastore/src/main/java/com/google/cloud/datastore/StructuredQuery.java

```diff
@@ -502,6 +505,46 @@ public static PropertyFilter eq(String property, Blob value) {
       return new PropertyFilter(property, Operator.EQUAL, of(value));
     }
 
+    public static PropertyFilter neq(String property, Value<?> value) {
```

Comment:
need docs/annotation

Reply:
Is there another place where I should add doc to? Looks like other operator methods in the context has no doc as well.